### DISCUSSION TITLE
Welcome to FFXIVPetRenamer Discussions!

### DISCUSSION THREAD

**Discussion by Glyceri** [Glyceri/FFXIVPetRenamer#1] [Announcements]
## 👋 Hello Gamers!
  This is a place where you can ask about:
  * Future updates
  * Feature requests
  * Bugs and/or errors

**Comment by valkayrie** [comment]
hello! I was wondering if you've given any thought to adding syncing via Mare Synchronos, bypassing the manual process of share codes. so far one other user in the Mare Discord has expressed interest as well. there is precedent with support for the Honorific plugin having support added.

**Reply by Glyceri** [reply]
Hey Valkayrie, thank you for asking! I will give you the cookie cutter response because I get asked this relatively often. I imagined this plugin being able to interact with Mare since day one. However, implementing a plugin into Mare takes an incredible amount of continuous work from Floof specifically. We have been in contact and the consensus is that it will come eventually? Maybe? Possibly?

**Reply by valkayrie** [reply]
thanks for letting me know! I tried searching here and in the Mare discord, I only saw the one response from last summer.

**Reply by Glyceri** [reply]
Np. Most people reach out to me in DM's and not so much here or the Mare Discord, making demand seem a LOT less than it actually is.

**Reply by bashemi** [reply]
Actually just came here from the Mare Sync discord to ask the same lol. I'd _love_ having this synced too. The Mare dev said something about an API?

Either way I can just force my friends to import my names until then lol. love the plugin btw, thanks for the hard work

**Reply by Glyceri** [reply]
Thanks for the support! I appreciate that a lot. The API is already there. Basically my side of the plogon is completely ready for mare. We are in talks of getting sync set up, but they have been just that, talks for now. It will come, hopefully, eventually, preferably sooner.

**Comment by midorifox** [comment]
Somewhat recently, Glamourer added the ability to apply automation to Battle NPCs (particularly, Trust Avatars) - in that we can apply appearance changes, but their name will always remain as something like "G'raha Tia's Avatar". Would it be possible to expand this plugin to allow Trust NPCs to be modified too?

**Reply by Glyceri** [reply]
I don't plan on adding that currently (I might do it later). I've been asked this a couple times and in my opinion it goes slightly beyond the scope of this plugin. Also it'd be a pain to add and requires extensive rewrites of a lot of things. If the demand becomes big I'll add it, so this comment definitely contributes.
